src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,59 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
 
+        writeResultFile(toFileName, readAndCalculatedFile(fromFileName));
```

Comment:
This method combines two operations: reading from one file and writing to another. According to the checklist, these operations should be performed in separate methods. Consider refactoring to separate the concerns.